infra/aws/terraform/prow-build-cluster/modules/cluster-autoscaler/cluster-autoscaler.tf

```diff
@@ -345,7 +345,7 @@ resource "kubernetes_manifest" "deployment_kube_system_cluster_autoscaler" {
       "namespace" = "kube-system"
     }
     "spec" = {
-      "replicas" = 1
+      "replicas" = 0
```

Comment:
Let's revert this to 1, cluster-autoscaler shouldn't delete instances detached from ASG.